src/mlpack/methods/softmax_regression/softmax_regression_function.hpp

```diff
@@ -176,7 +175,7 @@ class SoftmaxRegressionFunction
     return initialPoint.n_cols;
   }
   /**
-   * Return the number of separable functions 
+   * Return the number of separable functions
       (the number of predictor points).
```

Comment:
```suggestion
   * Return the number of separable functions (the number of predictor points).
```
Looks like this could fit on one line too.